xroad-catalog-lister/README.md

```diff
@@ -10,6 +10,49 @@ A class diagram illustrating X-Road Catalog Lister implementation with the `defa
 See also the [Installation Guide](../doc/xroad_catalog_installation_guide.md) and
 [User Guide](../doc/xroad_catalog_user_guide.md).
 
+## Configuration
+
+X-Road Catalog Lister can be configured by creating a copy of the [application.yaml](/src/main/resources/application.yaml) file.
```

Comment:
The `application.yaml` file should should only contain properties that the user is expected to change, e.g., `spring.datasource.username`, `spring.datasource.password`, `xroad-catalog.shared-params-file`. Now the file contains properties that the user should not change, e.g., `spring.jpa.*`. It would be better if the default configuration was included in the application JAR file and the user could override the properties that need to be overridden using the `application.yaml` file. In that way, the user don't need to mind about the other properties. The same comment applies to the lister and collector modules.

Reply:
I split the configurations section into different groups, `mandatory`, `optional`, and `fixed-values in application.yaml`. I hope this clarifies the configurations and meets your suggestion.

I left the `Required` column as some of these configurations parameters are conditionally mandatory. But it can be removed, if required.

Reply:
@raits and @petkivim , since the latest structure of the document is now splitting the configurations into mandatory and optional sections, I removed the `Required` column from the READMEs